doc/manual/variables-and-scoping.rst

```diff
@@ -296,15 +448,21 @@ block without creating any new bindings:
 Since ``let`` introduces a new scope block, the inner local ``x``
```

Comment:
You can create two kinds of bindings with `let` -- as part of the `var = value` declarations, or as part of the `code`. Do they behave the same? Which of these introduces new bindings?